src/hooks/useSegmentAnalytics.js

```diff
@@ -0,0 +1,57 @@
+export default function useSegmentAnalytics() {
+  !(function () {
```

Comment:
why do you need a self  invoking anonymous function? i never liked the syntax of this shit.

Reply:
part of the docs for segment, just straight copy pasted. Not sure how else we could go about it